public/app.js

```diff
@@ -3,17 +3,29 @@
 
     angular.module('OpenFinD3FC', [
         'ngRoute',
-        'openfin.showcase'
+        'openfin.thumbnails', 
+        'openfin.search',
+        'openfin.stock'
     ])
     .config(['$routeProvider', function($routeProvider) {
         $routeProvider
             .when('/', {
                 redirectTo: 'stocks'
             })
-            .when('/stocks', {
-                templateUrl: 'stocks.html',
-                controller: 'ShowcaseCtrl',
-                controllerAs: 'showcaseCtrl'
+            .when('/stocks/', {
+                templateUrl: 'thumbnails.html',
+                controller: 'ThumbnailsCtrl',
+                controllerAs: 'thumbnailsCtrl'
+            })
+            .when('/stock/:stock', {
+                templateUrl: 'stockData.html',
+                controller: 'StockCtrl',
+                controllerAs: 'stockCtrl'
+            })
+            .when('/search/:query', {
+                templateUrl: 'search.html',
+                controller: 'SearchCtrl',
+                controllerAs: 'searchCtrl'
             })
```

Comment:
Maybe make directory structure match your routes? This can provide some logic? This is suggestion for the future, not necessary this PR.